policies/apt.go

```diff
@@ -145,7 +145,13 @@ func aptChanges(aptInstalled, aptRemoved, aptUpdated []*osconfigpb.Package) erro
 		logger.Infof("Installing packages %s", changes.packagesToInstall)
 		if err := packages.InstallAptPackages(changes.packagesToInstall); err != nil {
 			logger.Errorf("Error installing apt packages: %v", err)
-			errs = append(errs, fmt.Sprintf("error installing apt packages: %v", err))
+
```

Comment:
i would have loved to the some unit test testing this fallback logic. I am actually working on adding more coverage to this package.

Reply:
Thanks. Now the fallback logic is moved here so that we can test this part.